configuration/concepts/program_attribute_types.csv

```diff
@@ -22,16 +22,19 @@ bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,
 2484cdc7-a77a-4305-8bcc-b9c90f384bf6,,,Cardiology,Cardiology,,,,,Misc,N/A,,,,,,,,,,,
 dc04cfe1-29d1-46a7-a0cf-b8baacfdbb31,,,Dermatology,Dermatology,,,,,Misc,N/A,,,,,,,,,,,
 13aa8824-8a29-46c9-b80e-9201f03e82e5,,,Wound,Wound,,,,,Misc,N/A,,,,,,,,,,,
+d0e2b959-bd5d-4547-a545-d0e3e4b417de,,,Needs Clearance,Needs Clearance,,,,,Misc,N/A,,,,,,,,,,,
 589b55a5-40e9-4b7c-8af6-1415b2c6a0bf,,,Needs Surgical Optimization,Needs Surgical Optimization,,,,,Misc,N/A,,,,,,,,,,,
+ef83d1d9-d355-4356-9b60-61217f9dc730,,,Cleared for Surgery,Cleared for Surgery,,,,,Misc,N/A,,,,,,,,,,,
```

Comment:
@basrem Is it necessary to add this here?
I see they are already in this [file](https://github.com/globalhealthcoalition/openmrs-config-ghc/blob/master/configuration/concepts/program_states.csv)